src/d3a/models/myco_matcher/__init__.py

```diff
@@ -0,0 +1,31 @@
+from d3a.constants import DOUBLE_SIDED_ALGO
+from d3a.models.myco_matcher.pay_as_bid import PayAsBidMatch
+from d3a.models.myco_matcher.pay_as_clear import PayAsClear
+
+
+class MycoMatcher:
+    def __init__(self):
+        self.bid_offer_pairs = []
+        if DOUBLE_SIDED_ALGO == 1:
+            self.match_algo = PayAsBidMatch()
+        elif DOUBLE_SIDED_ALGO == 2:
+            self.match_algo = PayAsClear()
+
+    @staticmethod
+    def sorting(obj, reverse_order=False):
+        if reverse_order:
+            # Sorted bids in descending order
+            return list(reversed(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate)))
+
+        else:
+            # Sorted bids in ascending order
+            return list(sorted(
+                obj.values(),
+                key=lambda b: b.energy_rate))
```

Comment:
```suggestion
def sorting(obj, reverse_order=False):
    # Sorted bids in descending order
    return sorted(
        obj.values(),
        key=lambda b: b.energy_rate, reverse=reverse_order)
```

Reply:
I would also suggest renaming the method to something different (as `sorting` isn't an imperative verb). For example `sort_by_energy_rate()`.

Reply:
It no more has a usage, I will remove it and adapt your suggestion [here](https://github.com/gridsingularity/d3a/pull/1121#discussion_r620240996)